Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Cursors/CursorModifier.cs

```diff
@@ -168,7 +165,7 @@ public Vector3 GetModifiedPosition(IMixedRealityCursor cursor)
             }
 
             FocusDetails focusDetails;
-            if (InputSystem.FocusProvider.TryGetFocusDetails(cursor.Pointer, out focusDetails))
+            if (MixedRealityManager.InputSystem.FocusProvider.TryGetFocusDetails(cursor.Pointer, out focusDetails))
```

Comment:
Not checking for no Input system
```
MixedRealityManager.InputSystem?.
```
Update and check throughout

Reply:
Fixed